apps/rock-paper-scissors/src/pages/home/Home.tsx

```diff
@@ -0,0 +1,305 @@
+import { useState, useMemo, useEffect } from 'react';
+import { /* useWasm, */ useMsToTime, useGetState } from 'hooks';
+import { useForm } from 'utils';
+import { ApiLoader } from 'components';
+import { SVGType } from 'types';
+import {
+  useAccount,
+  useCreateHandler,
+  useMetadata,
+  useSendMessage
+} from '@gear-js/react-hooks';
+import { blake2AsHex } from '@polkadot/util-crypto';
+import { stringToU8a, } from '@polkadot/util';
+import { Hex } from '@gear-js/api';
+
+import { Create } from './create';
+import { Start } from './start';
+import { Game } from './game';
+import { Details } from './details';
+import { Join } from './join'
+import { JoinDetails } from './joinDetails';
+import { Move } from './move';
+import { ReactComponent as GearBgSVG } from '../../assets/images/backgrounds/gearBG.svg'
+import { ReactComponent as LizardBgSVG } from '../../assets/images/backgrounds/lizard.svg'
+import { Reveal } from './reveal';
+import { RoundResult } from './round-result';
+import { GameResult } from './game-result';
+
+import metaAssets from '../../assets/metaWasm/rock_paper_scissors.meta.wasm'
+
+// function useCreateRockPaperScissors() {
+//   const { codeHash, meta } = useWasm();
+//   return useCreateHandler(codeHash, meta);
+// }
+
+// function useMessage(programID: Hex) {
+//   const { meta } = useWasm();
+//   return useSendMessage(programID, meta)
+// }
+//  temporary 
+
+function useCreateRockPaperScissors() {
+  const newHesh = process.env.REACT_APP_CODE_ADDRESS as Hex
+  const { metadata } = useMetadata(metaAssets)
+  return useCreateHandler(newHesh, metadata);
+}
+
+function useMessage(programID: Hex) {
+  return useSendMessage(programID, metaAssets)
+}
```

Comment:
```suggestion
function useCreateRockPaperScissors() {
  const newHesh = process.env.REACT_APP_CODE_ADDRESS as Hex
  const { metadata } = useMetadata(metaAssets)
  return useCreateHandler(newHesh, metadata);
}

function useRockPaperScissorsMessage(programID: Hex) {
  return useSendMessage(programID, metaAssets)
}
```

All hooks should be in hooks directory